management/backup.py

```diff
@@ -351,6 +351,33 @@ def list_target_files(config):
 	if p.scheme == "file":
 		return [(fn, os.path.getsize(os.path.join(p.path, fn))) for fn in os.listdir(p.path)]
 
+	elif p.scheme == "rsync":
+		env = load_environment()
+		rsync_fn_size_re = re.compile(r'.*    ([^ ]*) [^ ]* [^ ]* (.*)')
+		rsync_target = '{host}:{path}'
+
+		_, target_host, target_path = config['target'].split('//')
+		target_path = '/' + target_path
+
+		rsync_command = [ 'rsync',
+					'-e',
+					'/usr/bin/ssh -o StrictHostKeyChecking=no -oBatchMode=yes',
+					'--list-only',
+					'-r',
+					rsync_target.format(
+						host=target_host,
+						path=target_path)
+				]
+
+		code, listing = shell('check_output', rsync_command, get_env(env), trap=True)
```

Comment:
`get_env(env)` is unnecessary here -- that's only intended for setting up the environment for duplicity. That also means `env = load_environment...` can be dropped higher up.